concourse/scripts/concourse-lite-environment.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+PROJECT_DIR=$(cd "${SCRIPT_DIR}/../.." && pwd)
+
+hashed_password() {
+  echo "$1" | shasum -a 256 | base64 | head -c 32
+}
+
+DEPLOY_ENV=${1:-${DEPLOY_ENV:-}}
+if [ -z "${DEPLOY_ENV}" ]; then
+  echo "Must specify DEPLOY_ENV as \$1 or environment variable" 1>&2
+  exit 1
+fi
+
+AWS_ACCOUNT=${AWS_ACCOUNT:-dev}
+
+CONCOURSE_URL="${CONCOURSE_URL:-http://localhost:8080}"
+FLY_TARGET="${FLY_TARGET:-${DEPLOY_ENV}-bootstrap}"
+FLY_CMD="${PROJECT_DIR}/bin/fly-bootstrap"
+
+CONCOURSE_ATC_USER=${CONCOURSE_ATC_USER:-admin}
+if [ -z "${CONCOURSE_LITE_ATC_PASSWORD:-}" ]; then
+    CONCOURSE_LITE_ATC_PASSWORD=$(hashed_password "$(gpg --export-secret-key -a)")
```

Comment:
It feels uncomfortable using my private key in this way. I can't describe the exact problem, but it *feels* wrong. We had similar concerns about `AWS_ACCESS_KEY` in the very first implementation.

The original PR used a less secure predictable string on the basis that the bootstrap Concourse can only be accessed using an SSH tunnel. Could you describe the rationale for changing it?